login-workflow/src/components/PasswordTextField/PasswordTextField.tsx

```diff
@@ -14,7 +14,7 @@ import VisibilityOff from '@mui/icons-material/VisibilityOff';
  * @category Component
  */
 export const PasswordTextField: React.FC<React.PropsWithChildren<TextFieldProps> & { icon?: ReactNode }> = (props) => {
-    const { icon } = props;
+    const { icon, sx } = props;
```

Comment:
We should probably do something like 
```suggestion
    const { icon, sx, ...otherProps } = props;
```
and then pass the `otherProps` to the component. Otherwise you're passing sx twice